apps/web-giddh/src/app/multi-currency-reports/filter/filter-multi-currency.component.ts

```
@@ -0,0 +1,297 @@
+    public selectedDate(value: any) {
+        this.filterForm.controls['from'].setValue(dayjs(value.picker.startDate).format(GIDDH_DATE_FORMAT));
+        this.filterForm.controls['to'].setValue(dayjs(value.picker.endDate).format(GIDDH_DATE_FORMAT));
+    }
+
+    public ngAfterViewInit() {
+        this.cd.detectChanges();
+    }
+
+    public filterData() {
+        this.onPropertyChanged.emit();
+        let a = this.search = '';
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)